packages/client/src/client/components/Typography.tsx

```diff
@@ -15,5 +15,6 @@ export const Typography = styled.div<TypographyProps>`
     color ? theme.newTheme.color[color] : "inherit"};
 
   margin-block-end: 0;
-  ${({ allowLineHeight }) => (allowLineHeight ? undefined : `line-height: 1`)};
+  ${({ allowLineHeight }) =>
```

Comment:
ah, noted .. hopefully we will be able to remove our line-height hacks in due course